vars/starter_dev.yml

```diff
@@ -0,0 +1,34 @@
+---
+drupal_build_composer_project: false
+drupal_deploy: true
+# XXX: Just pointing at my repo for the moment, for testing... should be pointed
+# back at Islandora's prior to merging.
+drupal_deploy_repo: https://github.com/adam-vessey/islandora-starter-site.git
+# XXX: Should be made to point at "main".
+drupal_deploy_version: implementation
```

Comment:
Yeah, needs to be pointed at the merged bits, whenever it's possible to do so.

Reply:
Done in https://github.com/Islandora-Devops/islandora-playbook/pull/226/commits/a18bde06eaa5a4b0e54c3b87b6e400a716184c1d